src/psyclone/tests/psyir/backend/opencl_test.py

```diff
@@ -0,0 +1,182 @@
+# -----------------------------------------------------------------------------
```

Comment:
This file has a couple of pylint errors.

Reply:
fixed